tests/integration_test.py

```diff
@@ -181,7 +181,8 @@ def runTest(self):
             container = self.client.create_container(
                 'busybox',
                 ['ls', mount_dest], volumes={mount_dest: {}},
-                host_config=create_host_config(binds=binds)
+                host_config=create_host_config(
+                    binds=binds, network_mode='none')
```

Comment:
Why are we setting this explicitly here if it's already handled in `create_container_config`?